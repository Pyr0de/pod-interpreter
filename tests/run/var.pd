let a = 10;
let b = -10;
let c = 10 * 10;
let d = "abc" + "efg";
let e = false;

print a;
print b;
print c;
print d;
print e;

a = 12;
print a;
b = "abc";
print b;

print z;
print a;
