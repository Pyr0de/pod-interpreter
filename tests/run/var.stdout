ExitCode: 66
10
-10
100
abcefg
false
12
abc
